src/ontology/mondo-edit.obo

```diff
@@ -93312,11 +93312,14 @@ def: "A chronic inflammatory genetically determined disease of the skin marked b
 subset: otar {source="MONDO:OTAR"}
```

Comment:
the definition of MONDO:004980 needs to be revised, it does not currently match what is in MESH:
https://www.ncbi.nlm.nih.gov/medgen/?term=D003876

The def should be:
A common chronic pruritic inflammatory skin disease with a strong genetic component. Onset typically occurs during the first 2 years of life.
xref:OMIM:603165

(MESH is pulling this def from OMIM, so it's best to xref OMIM directly)

Reply:
updated

Reply:
did you update the def? I don't see it on the diff

Reply:
I was not able to push the changes until after meetings.